### DISCUSSION TITLE
Verifying account while creating new organization is not working

### DISCUSSION THREAD

**Discussion by Zebiano** [community/community#24016] [New to GitHub]
<p>Hey everyone!</p>
<p>I’m not being able to verify my account while creating a new organization through the following link: <a href="https://github.com/account/organizations/new?coupon=&amp;plan=team_free" rel="noopener nofollow ugc">https://github.com/account/organizations/new?coupon=&amp;plan=team_free</a></p>
<p>The “Verify your account” box is simply blank and I cannot make it load. I do see the mouse changing to “click” on specific coordinates of said box, but I have no idea what I have to do in order to continue. I’ve tried randomly clicking, but without any success.</p>
<p>Here’s an image of what I see:<br>
<img src="https://user-images.githubusercontent.com/9255144/181080642-1f27282f-75b0-4894-ab15-2b760a87b3d2.png" alt="image" data-base62-sha1="lgyCkHXcGRjJPBaHiJfpZttYXaL" width="564" height="468"></p>
<p>Any help would be appreciated!<br>
Zeb</p>

**Comment by byrneh** [comment, marked as answer]
<p>Hi <a class="mention" href="/zebiano">@Zebiano</a>,welome.<br>
There are some standard browser things you can try yourseelf if not already done so.<br>
Such as ensuring your browser is up to date, clearing browser cache. restarting machine, trying a different browser., trying incognito mode in browser , these will help resolve or help further narrow down the area of the problem.</p>
<p>If no success, it will probably need you to supply more details, including the basics like what OS,  browser and version you are experiencing the problem with</p>

**Comment by Zebiano** [comment]
<p>Hey!<br>
After playing around with some of your suggestions, this is what I’ve found:</p>
<ol>
<li>Edge worked</li>
<li>Incognito on Chrome worked</li>
<li>Chrome kept on not working, even after clearing cache and cookies.</li>
</ol>
<p>I don’t think my extensions could be causing problems.</p>
<p>I’m guessing this is a problem only I am encountering at the moment, which makes me believe there’s something wrong with my machine and not the GitHub website.</p>
<p>I’m down for some debugging if any GH Dev is interested, but other than that I’ve fixed my problem for now so I’m marking this answer as the solution.</p>
<p>Thank you for the help!<br>
Zeb</p>

**Comment by byrneh** [comment]
<p>Hi <a class="mention" href="/zebiano">@Zebiano</a> well the most important  thing is you are up and running, so that is good news to hear.</p>
<p>Edge and Chrome are similar and share common codebase if you do get around to doing a comparison and some debugging.</p>

**Comment by ItsBrank** [comment]
Having this issue right now, can't create an org because when I do the verification it just says "something went wrong" and the reload button makes the entire thing just a blank white square.

**Reply by Antonio-Riccelli** [reply]
Same here. Tried different browsers, no luck. Did you fix it, @ItsBranK ?

**Comment by ChinSekYi** [comment]
it's 2024 and I still have the same issue. Tried chrome, incognito, edge, but none worked. Tried it on my ipad and it doesn't work to. Anyone else have a solution for this?

**Reply by 3ein39** [reply]
> it's 2024 and I still have the same issue. Tried chrome, incognito, edge, but none worked. Tried it on my ipad and it doesn't work to. Anyone else have a solution for this?

+1

**Reply by zulonas** [reply]
+1

**Reply by gojushin** [reply]
+1

**Reply by ghdyd586** [reply]
+1, Tried Chrome, Chrome incognito, Safari, and Safari private mode. None of them did work.

update
- Safari Mobile didn't work

**Reply by Antonio-Riccelli** [reply]
> it's 2024 and I still have the same issue. Tried chrome, incognito, edge, but none worked. Tried it on my ipad and it doesn't work to. Anyone else have a solution for this?

Same. Looks like it's broken. Tried Safari as well and no luck.

**Reply by zulonas** [reply]
I was able to successfully verify my account. It seems there was a temporary bug or some issue with the verification system.

<img width="758" alt="image" src="https://github.com/user-attachments/assets/1d2470b1-c55b-4a96-9878-faa69cedfa01">

**Reply by stevenpetein** [reply]
Hi folks
found this one, having the same issue.
Tried different browsers, incognito, couple days in between.
No success so far.. Nothing happening after first check..

**Reply by isleofwighter** [reply]
Hi. I do not get the big green tick but can click the green Next which then changes to Setting up Account but nothing happens, gave it 30 minutes then gave up. Happened twice. In Windows 11 (fully updated, Firefox fully updated, have cleared cache and all the usual checks ?????

**Reply by isleofwighter** [reply]
Hi again. I did the captcha and but got a message unable to verify captcha. I am not a company I am doing this on my private desktop, no firewall, not via a vpn. Tried to load the captcha test page and got a black page. but the load test worked fine?
